index-pl.html

```diff
@@ -206,61 +206,61 @@ <h2>Why Masks Protect Us Twice</h2>
     </p>
 
     <p>
-      At the other extreme, if <b>everyone</b> wore a mask, then whenever two people meet, the chance that they <span class="yellow">both wear masks</span> is 100%.
+      W przypadku drugiej skrajności, gdy założymy, że <b>wszyscy</b> noszą maski, w przypadku spotkania dwojga ludzi, szansa, że <span class="yellow">oboje mają założone maski,</span> wynosi 100%.
     </p>
 
     <network :maskusage="1"></network>
 
     <p>
-      In this case, we’d only have to consider the <span class="yellow">last route</span> of disease transmission.
-      Assuming masks are 50% effective in each direction, the population would see a <span class="yellow">75% drop</span> in disease transmission.
+      W tym przypadku więliśmy pod uwagę jedynie <span class="yellow">ostatnią drogę</span> rozprzestrzeniania się choroby.
+      Zakładając, że maski mają 50-procentową skutechność w każdym kierunku, w populacji zaobserwoalibyśmy<span class="yellow">75-procentowy spadek</span> w roznoszeniu choroby.
     </p>
 
     <p>
-      So when <i>everyone</i> wears a mask (or when <i>no one</i> wears one), it's straightforward to calculate the drop in disease transmission in the population, because there's only one route involved.
+      Tak więc, gdy <i>wszyscy</i> noszą maski (lub gdy <i>nikt</i> ich nie nosi), łatwo jest obliczyć spadek w transmisji choroby pośród populacji, ponieważ tylko jedena ze skrajności jest uwzględniona.
     </p>
 
     <p>
-      But in reality, some people wear masks and others don’t.
-      Which means the virus can spread through a mix of all four routes.
-      How likely each route is will depend on how many people wear masks.
+      W rzeczywistości jednak, niektórzy ludzie noszą maski a inni nie.
+      To oznacza, że wirus może rozprzestrzeniać się poprzez kombinację wszystkich czterech dróg rozprzestrzeniania.
+      Prawdopodobieństwo każdej z nich jest podyktowane tym jak wiele ludzi nosi maski.
     </p>
 
     <label class="row half bigsplash">
-      <div>Mask Usage: <span class="dotted">{{convertToPercent(pnetwork)}}</span>%</div>
+      <div>Użycie Masek: <span class="dotted">{{convertToPercent(pnetwork)}}</span>%</div>
       <slider v-model="pnetwork" :min="0" :max="1" :step="0.01"></slider>
     </label>
 
     <network :maskusage="pnetwork"></network>
 
     <p>
-      For example, <b>if 50% of people wear masks</b>, then whenever two people meet at random, the chance that <span class="yellow">both people wear masks</span> is 50% ⨉ 50%, i.e. 25%.
-      Similarly, you can work out the chance of the other three disease transmission routes.
+      Na przykład, <b>jeśli 50% ludzi nosi maski</b>, zawsze gdy spotyka się dwoje przypadkowych ludzi, szansa na to, że <span class="yellow">oboje mają maski</span> wynosi 50% ⨉ 50%, czyli 25%.
+      W podobny sposób możemy obliczyć szanse na zaistnienie pozostałych trzech dróg na rozprzestrzenianie się choroby.
```

Comment:
zaistnienie zastąpić słowem "wystąpienie" ?
a trzech dróg rozprzestrzeniania się choroby -> trzech możliwości rozprzestrzeniania się choroby ?

Reply:
a trzech dróg rozprzestrzeniania się choroby -> trzech możliwości rozprzestrzeniania się choroby ?

z tym jest trochę problem, bo tekst się później odnosi do tych dróg/możliwości i mozliwości nie zawsze pasuje. 

Zastanówmy się jeszcze jak najlepiej przetłumaczyć: disease transmission routes. 
Bo to się wszędzie przewija często.